.travis.yml

```diff
@@ -10,8 +10,16 @@ node_js:
 - '5'
 - '4'
 - '0.12'
-script: make test-ci
+script: 
+  - 'if [ -n "${LINT-}" ]; then make lint ; fi'
+  - 'make test-ci'
```

Comment:
this will run the tests even in the linter run - i think you want `if [ -z "${LINT-}" ]` here to skip this.